src/os/xlitob.c

```diff
@@ -50,9 +46,7 @@ void _Litob(_Pft *args, char type) {
     }
 
     if (args->prec < 0 && (args->flags & (FLAGS_ZERO | FLAGS_MINUS)) == FLAGS_ZERO) {
-        i = args->width - args->n0 - args->nz0 - args->n1;
-
-        if (i > 0) {
+        if ((i = args->width - args->n0 - args->nz0 - args->n1) > 0) {
```

Comment:
was this kind of change necessary to match the file for iQue? I'd prefer to keep unnecessary changes out. though if this is how ultralib did it, I guess that's fine